src/main/java/org/snomed/snowstorm/core/data/domain/Concept.java

```diff
@@ -240,6 +244,11 @@ public Concept addIdentifier(Identifier identifier) {
 		return this;
 	}
 
+	public Concept addAnnotation(Annotation annotation) {
+		annotations.add(annotation);
+		return this;
+	}
+
```

Comment:
@QuyenLy87 this looks like it will throw a NPE depending on which constructor is used to instantiate the Concept.

Reply:
Hi @dmcgihtsdo this is not the issue. The addAnnotation is similar to other methods (for example: addDescription, addRelationship, addAxiom...) When instantiating the concept, we always call the default constructor "public Concept()". So the property "annotations" will never be null.